src/main/java/frc/robot/subsystems/StateMachine.java

```diff
@@ -271,6 +291,8 @@ public static enum TargetState {
     PREP_CORAL_L4,
     PREP_NET,
     PREP_PROCESSOR,
+    ALGAE_PREP_0,
+    CORAL_PREP_0,
```

Comment:
Could this be PREP_ALGAE_ZERO and PREP_CORAL_ZERO to better match the existing names?